bin/create.js

```diff
@@ -83,6 +83,11 @@ module.exports = Cli.createCommand('create', {
                 description: 'Allow the webtask server to cache code. When disabled, your code will be loaded on each request by the webtask runtime environment, introducing additional latency. Leaving this disabled is useful while developing.',
                 type: 'boolean',
             },
+            'minify': {
+                description: 'Generate a minified production build',
+                type: 'boolean',
+            },
```

Comment:
Please add:
```
dest: 'minify',
```

This will assign that param to the `minify` attribute of `args` thereby keeping the rest of the PR intact.

I would like all _bundling_-related arguments prefixed with `--bundle`.

Thanks for the PR!